projects/steering/src/steering_can.c

```diff
@@ -7,16 +7,18 @@
 #include "steering_digital_input.h"
 #include "steering_events.h"
 
+static bool enable_regen_braking = true;

```

Comment:
Nitpick: the [coding standards](https://uwmidsun.atlassian.net/wiki/spaces/ELEC/pages/9306218/Coding+Standards) say that static variables should be prefixed with `s_`, so this should be `s_enable_regen_braking`. (Sorry I know I gave you the variable name!)